### PR TITLE
Samples page with processing information

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

https://github.com/AlexsLemonade/refinebio-frontend/issues/702

## Purpose/Implementation Notes

This adds an internal samples page that we can use to check sample processing information.

## Types of changes

* [x] New feature (non-breaking change which adds functionality)

## Functional tests

None

## Checklist

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)
* [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

image

### CODE REVIEW

**Comment by cgreene** [comment]
I'm a HUGE fan of the direction this is going in! It will help members of our science team respond to experiment requests as well.

**Comment by arielsvn** [comment]
This should be ready for review @davidsmejia. If you want to test it, you'll need to run a local version of the API since we haven't deployed the latest API changes.

I added a bit more information to the page and collapsed all the jobs table in an accordion.

image

File: src/pages/Sample/SampleDebugContext.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  return {
+    sample,
+    originalFiles: originalFiles.results,
+    computedFiles: computedFiles.results,
+    jobs: mergeJobs(processorJobs.results, downloaderJobs.results, runningJobs),
+  };
+}
+
+export async function getOriginalFiles(sampleId) {
```
Comment (davidsmejia) [inline src/pages/Sample/SampleDebugContext.js]: Would these functions be better off in `./api/computedFiles.js` etc.

**Comment by arielsvn** [comment]
Thanks for reviewing @davidsmejia. Moved the functions into `./api` in case you want to take another look.

**Pull request merged**